pkgs/development/python-modules/pymemoize/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "PyMemoize";
```

Comment:
Please normalize `pname` like `pymemoize`.
Note that the url for pypi includes `PyMemoize` so you'll have to do
```
pname = "PyMemoize";
```
in the `fetchPypi` call.

Reply:
fixed